main.go

```diff
@@ -22,17 +22,21 @@ var (
 
 // facilitates CLI arguments
 func init() {
-	if os.Getenv("GITHUB_USER") == "" {
-		fmt.Fprintln(os.Stderr, "Get a Github personal access token and create an environment variable GITHUB_USER and try again.")
-		fmt.Println("You can create one right here https://github.com/settings/tokens")
-	}
+
 	repoURL = flag.String("repo", "", "a string") // the project
 	title = flag.String("title", "", "a string")
 	desc = flag.String("desc", "", "a string")
 	flag.Parse()
 }
 
 func main() {
```

Comment:
How about keeping it in the init function?
Should we follow [this](https://golang.org/doc/effective_go.html#init) or remove the init function completely?